pyproject.toml

```diff
@@ -191,7 +191,11 @@ sdist.include = ["fast_pauli/__version__.py"]
 write_to = "fast_pauli/__version__.py"
 
 [tool.cibuildwheel]
-build = ["cp310-manylinux_x86_64", "cp311-manylinux_x86_64", "cp312-manylinux_x86_64", "cp313-manylinux_x86_64"]
+# build = ["cp310-manylinux_x86_64", "cp311-manylinux_x86_64", "cp312-manylinux_x86_64", "cp313-manylinux_x86_64"]
 build-frontend = "build"
 test-requires = "pytest"
 test-command = "pytest -s -vv --import-mode importlib {project}/tests/fast_pauli"
+
+# [tool.cibuildwheel.macos]
+before-build = "brew install llvm libomp"
```

Comment:
you might need to have brew installed first. macos is not shipped with it installed

Reply:
runners preinstalled packages can be found here:  https://github.com/actions/runner-images/blob/main/images/macos/macos-13-arm64-Readme.md

Reply:
oh that's nice, resolved